tucson-teststand/AT/docker-compose.yaml

```diff
@@ -0,0 +1,45 @@
+version: "3.6"
+
+services:
+
+  atmcs-sim:
+    image: ts-dockerhub.lsst.org/atmcs_sim:${CYCLE}
+    container_name: atmcs-sim
+    environment:
+      - LSST_DDS_PARTITION_PREFIX=${SITE_PARTITION_PREFIX}
+      - OSPL_MASTER_PRIORITY=0
+      - OSPL_INFOFILE=/tmp/ospl-info-atmcs.log
+      - OSPL_ERRORFILE=/tmp/ospl-error-atmcs.log
+    volumes:
+      - ${OSPL_SHMEM_SOCKET_DIR}:/tmp/
+      - ${OSPL_SHMEM_CONFIG}:${OSPL_MOUNT_POINT}
+    tty: true
+    ipc: host
+    pid: host
+    network_mode: host
```

Comment:
If you are using shmem mode, you don't need to construct this container with `host` network. Only `ipc`, `pid` and the volumes are needed...

Reply:
Done.